src/meson.build

```diff
@@ -260,7 +260,7 @@ if not opt_miniupnpc.disabled()
   endif
 endif
 
-if not opt_curses.disabled()
+if opt_curses.enabled() or (opt_curses.auto() and os_name != 'windows')
```

Comment:
Why disable it by default for Windows? This also doesn't seem related to OpenBSD sandboxing at all.

Reply:
To have fully green CI here.

- https://github.com/gsliepen/tinc/runs/6247115066#step:8:355
- https://github.com/gsliepen/tinc/runs/6248463823#step:8:335

Broken by a recent change to the `ncurses` package. Probably [this][commit] one. I'll look into it later.

[commit]: https://github.com/msys2/MINGW-packages/pull/11586